Vindicta.Altis/Templates/Factions/3CB_BAF.sqf

```diff
@@ -0,0 +1,143 @@
+_array set [T_REQUIRED_ADDONS, [
+	"rhs_c_troops",		// RHSAFRF
+	"rhsusf_c_troops"	// RHSUSAF
+]];
+
+//==== Infantry ====
+_inf = [];
+_inf resize T_INF_SIZE;
+_inf set [T_INF_DEFAULT, ["UK3CB_BAF_Rifleman_MTP"]];
+
+_inf set [T_INF_SL, ["UK3CB_BAF_SC_MTP"]];
+_inf set [T_INF_TL, ["UK3CB_BAF_FT_MTP", "UK3CB_BAF_FT_762_MTP"]];
+_inf set [T_INF_officer, ["UK3CB_BAF_Officer_MTP"]];
+_inf set [T_INF_GL, ["UK3CB_BAF_Grenadier_MTP", "UK3CB_BAF_Grenadier_762_MTP"]];
+_inf set [T_INF_rifleman, ["UK3CB_BAF_Rifleman_MTP"]];
+_inf set [T_INF_marksman, ["UK3CB_BAF_Marksman_MTP"]];
+_inf set [T_INF_sniper, ["UK3CB_BAF_Sniper_MTP_Ghillie_L115", "UK3CB_BAF_Sniper_MTP_Ghillie_L135"]];
+_inf set [T_INF_spotter, ["UK3CB_BAF_Spotter_MTP_Ghillie_L129", "UK3CB_BAF_Spotter_MTP_Ghillie_L85"]];
+_inf set [T_INF_exp, ["UK3CB_BAF_Explosive_MTP_REC"]];
+_inf set [T_INF_ammo, ["UK3CB_BAF_MATC_MTP", "UK3CB_BAF_Rifleman_762_MTP"]];
+_inf set [T_INF_LAT, ["UK3CB_BAF_LAT_ILAW_MTP", "UK3CB_BAF_LAT_ILAW_762_MTP"]];
+_inf set [T_INF_AT, ["UK3CB_BAF_LAT_MTP", "UK3CB_BAF_LAT_762_MTP"]];
+_inf set [T_INF_AA, ["UK3CB_BAF_MAT_MTP"]];
+_inf set [T_INF_LMG, ["UK3CB_BAF_MGLMG_MTP"]];
+_inf set [T_INF_HMG, ["UK3CB_BAF_MGGPMG_MTP"]];
+_inf set [T_INF_medic, ["UK3CB_BAF_Medic_MTP"]];
+_inf set [T_INF_engineer, ["UK3CB_BAF_Engineer_MTP"]];
+_inf set [T_INF_crew, ["UK3CB_BAF_Crewman_MTP"]];
+_inf set [T_INF_crew_heli, ["UK3CB_BAF_HeliCrew_MTP"]];
+_inf set [T_INF_pilot, ["UK3CB_BAF_HeliPilot_RAF_MTP"]];
+_inf set [T_INF_pilot_heli, ["UK3CB_BAF_HeliPilot_Army_MTP"]];
+//_inf set [T_INF_survivor, [""]];
+//_inf set [T_INF_unarmed, [""]];
+
+// Recon
+_inf set [T_INF_recon_TL, ["UK3CB_BAF_SC_MTP_REC"]];
+_inf set [T_INF_recon_rifleman, ["UK3CB_BAF_Pointman_MTP_REC"]];
+_inf set [T_INF_recon_medic, ["UK3CB_BAF_Medic_MTP_REC"]];
+_inf set [T_INF_recon_exp, ["UK3CB_BAF_Explosive_MTP_REC"]];
+_inf set [T_INF_recon_LAT, ["UK3CB_BAF_LAT_ILAW_MTP"]];
+//_inf set [T_INF_recon_marksman, ["rhs_vmf_recon_marksman", "rhs_vmf_recon_marksman_vss"]];
+_inf set [T_INF_recon_JTAC, ["UK3CB_BAF_FAC_MTP_REC"]];
+
+
+// Divers, still vanilla
+//_inf set [T_INF_diver_TL, ["B_diver_TL_F"]];
+//_inf set [T_INF_diver_rifleman, ["B_diver_F"]];
+//_inf set [T_INF_diver_exp, ["B_diver_exp_F"]];
+
+
+
+//==== Vehicles ====
+_veh = +(tDefault select T_VEH);
+_veh set [T_VEH_SIZE-1, nil];
+_veh set [T_VEH_DEFAULT, ["UK3CB_BAF_LandRover_Snatch_FFR_Green_A_MTP"]];
+
+_veh set [T_VEH_car_unarmed, ["UK3CB_BAF_LandRover_Soft_FFR_Green_A_MTP", "UK3CB_BAF_LandRover_Snatch_FFR_Green_A_MTP", "UK3CB_BAF_LandRover_Hard_FFR_Green_A_MTP"]];
+_veh set [T_VEH_car_armed, ["UK3CB_BAF_LandRover_WMIK_GMG_FFR_Green_A_MTP", "UK3CB_BAF_LandRover_WMIK_GPMG_FFR_Green_A_MTP", "UK3CB_BAF_LandRover_WMIK_HMG_FFR_Green_A_MTP"]];
+
+_veh set [T_VEH_MRAP_unarmed, ["UK3CB_BAF_LandRover_Hard_FFR_Green_A_MTP"]];
+_veh set [T_VEH_MRAP_HMG, ["UK3CB_BAF_Husky_Passenger_HMG_Green_MTP", "UK3CB_BAF_Husky_Passenger_GPMG_Green_MTP"]];

```

Comment:
In the future, there shouldn't be 6 MRAPs with HMGs pulling up to a town for one attack. If I'm not mistaken this had to do with the logistics version of this vehicle not having enough spots for the entire infantry squad and them having to get this many cars to fit in. Quote me if I am wrong.